### PR TITLE
[zh] Add glossary userns

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/reference/glossary/userns.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<!--
+In the context of user namespaces, the namespace is a Linux kernel feature, and not a
+{{< glossary_tooltip text="namespace" term_id="namespace" >}} in the Kubernetes sense
+of the term.
+-->
+在用户名字空间语境中，名字空间是 Linux 内核的功能特性而不是 Kubernetes 意义上的
+{{< glossary_tooltip text="名字空间" term_id="namespace" >}}概念。
+
+<!-- TODO: https://kinvolk.io/blog/2020/12/improving-kubernetes-and-container-security-with-user-namespaces/ -->
+
```
Comment (howieyuen) [inline content/zh/docs/reference/glossary/userns.md:56]: shall we keep the last hyper link here?

Comment (tengqm) [reply content/zh/docs/reference/glossary/userns.md:56]: This link is from English text, as a comment. We should keep it.

Comment (howieyuen) [reply content/zh/docs/reference/glossary/userns.md:56]: ok, that's fine

**Pull request merged**